src/deepqmc/wf/paulinet/paulinet.py

```diff
@@ -173,13 +176,17 @@ def __init__(
         )
         backflow_spec = {
             'orbital': [n_orbitals, backflow_channels],
-            'det': [max(n_up, n_down), len(self.confs) * backflow_channels],
+            'det': [n_up + n_down, len(self.confs) * backflow_channels],
         }[backflow_type]
         if backflow_transform == 'both':
             backflow_spec[1] *= 2
         self.backflow_type = backflow_type
         self.backflow_transform = backflow_transform
         self.backflow_op = backflow_op or BackflowOp()
+        coords = mol.coords
```

Comment:
If you don't mind I would refactor the dummy coords section, because this way it would introduce conflicts with the transferability branch.

Reply:
Sure. Feel free to just add a commit to the PR, I might then rebase